unit_tests/testVTUFiles/testVTUFiles.f90

```diff
@@ -138,10 +143,14 @@ SUBROUTINE testMultiVTU()
       CALL SetupTest2_Mesh()
       CALL testVTUFile%writeMesh(testVTKMesh)
       bool=testVTUFile%hasMesh
-      ASSERT(bool,'testVTUFile%writeMesh(...) UNSTRUCTURED_GRID 0')
+      ASSERT(bool,'testVTUFile%hasMesh')
+      INQUIRE(FILE='testPVTU_0.vtu',EXIST=bool)
```

Comment:
Be careful here. You should make sure the test deletes this file upon completion. Otherwise multiple executions of the test can lead to fall positives.